File: mobile/src/main/java/tr/xip/prayertimes/api/objects/City.java
package tr.xip.prayertimes.api.objects;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by ix on 11/30/14.
 */
public class City implements Serializable {

    @SerializedName("SehirId")
    private String id;

    @SerializedName("SehirAdi")
    private String name;

    public City(String id, String name) {
        this.id = id;
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: mobile/src/main/kotlin/tr/xip/prayertimes/model/City.kt
package tr.xip.prayertimes.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class City : Serializable {
    var databaseId: Int? = null
        internal set

    @SerializedName("SehirId")
    var id: String? = null

    @SerializedName("SehirAdi")
    var name: String? = null

    constructor(id: String, name: String) {
        this.id = id
        this.name = name
    }

    constructor(databaseId: Int, id: String, name: String) {
        this.databaseId = databaseId
        this.id = id
        this.name = name
    }
}
